### PR TITLE
Updated Geo-replication doc to only use Secure(non-root) setup

### PR DESCRIPTION
**Pull request by aravindavk**
Updates: https://github.com/gluster/glusterfs/issues/714
Signed-off-by: Aravinda Vishwanathapura <aravinda@kadalu.io>

### CODE REVIEW

**Comment by aravindavk** [comment]
@sunnyku @kotreshhr @harigowtham ^^

File: docs/Administrator Guide/Geo Replication.md [hunk 1]
```diff
@@ -38,192 +25,131 @@ geo-replication:
...
+Restart `glusterd` service on all Slave nodes.
 
 ## Setting Up the Environment for Geo-replication
 
-**Time Synchronization**
+### Time Synchronization
+
+On bricks of a geo-replication master volume, all the servers' time
+must be uniform. You are recommended to set up NTP (Network Time
+Protocol) service to keep the bricks sync in time and avoid the
```
Comment (kotreshhr) [inline docs/Administrator Guide/Geo Replication.md:113]: nit: NTP or similar service

File: docs/Administrator Guide/Geo Replication.md [hunk 2]
```diff
@@ -38,192 +25,131 @@ geo-replication:
...
-    12.20 hrs and brick 2 of the master is at 12.10 hrs with 10 minutes
-    time lag, all the changes in brick2 between this period may go
-    unnoticed during synchronization of files with Slave.
+### Password-less SSH
 
-**Password-less SSH**
 Password-less login has to be set up between the host machine (where
-geo-replication Create command will be issued) and one of the Slave node
+geo-replication Create command will be issued) and one of the Slave
+node.
```
Comment (kotreshhr) [inline docs/Administrator Guide/Geo Replication.md:125]: ...... and one of the Slave node for the unprivileged account created above.

File: docs/Administrator Guide/Geo Replication.md [hunk 3]
```diff
@@ -264,22 +177,20 @@ gluster volume geo-replication <master_volume> \
     create [ssh-port <port>] push-pem|no-verify [force]
```
Comment (kotreshhr) [inline docs/Administrator Guide/Geo Replication.md:177]: Remove the optional slave_user signature.
[<slave_user>@] to <slave_user>@

Same in all other places below.

File: docs/Administrator Guide/Geo Replication.md (hunk 1 above)
Comment (aravindavk) [reply docs/Administrator Guide/Geo Replication.md]: Done

File: docs/Administrator Guide/Geo Replication.md (hunk 2 above)
Comment (aravindavk) [reply docs/Administrator Guide/Geo Replication.md]: Done

File: docs/Administrator Guide/Geo Replication.md (hunk 3 above)
Comment (aravindavk) [reply docs/Administrator Guide/Geo Replication.md:177]: Done

**Comment by amarts** [comment]
> Are we also blocking the root geo-rep creation internally in the code?

I feel that would be a bad move for now. We can just add it as a WARNING in CLI if needed... otherwise, just after upgrade many setups would just fail, because their currently deployed root login based geo-replication is all failing after upgrade!

**Pull request merged**